lib/features/context-pad/ContextPadIcons.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * To change the icons, modify the SVGs in `./resources`, execute `npx svgo -f resources --datauri enc -o dist`,
+ * and then replace respective icons with the optimized data URIs in `./dist`.
+ */
+var icons = {
```

Comment:
Let's not expose icons as a default export, but through a dedicated export:

```
const append = '...';

export { append };
``` 

This way the file can be tree-shaken (only actual icons used are consumed).

CC @barmac.